### PR TITLE
[DAT-206] Add colombia cotization

### PR DESCRIPTION
**Pull request by nreal**
# Background
We need to add  Colombia currency to send to Sap system and Doppler

### CODE REVIEW

File: DopplerJobsServer/appsettings.development.json [hunk 1]
```diff
@@ -44,13 +44,13 @@
...
-    "CurrencyCodeList": [ "ARS" ],
+    "CurrencyCodeList": [ "ARS", "MXN", "COP" ],
     "InsertCurrencyQuery": "[dbo].[InsertNewCurrencyRate]",
     "HolidayRetryCountLimit": 5
   },
   "DopplerSapConfiguration": {
-    "CurrencyEndpoint": "http://localhost:5002/sap/billing/SetCurrencyRate",
-    "BillingEndpoint": "http://localhost:5002/Billing/CreateBillingRequest"
+    "CurrencyEndpoint": "http://localhost:5002/billing/SetCurrencyRate",
+    "BillingEndpoint": "http://localhost:5002/billing/CreateBillingRequest"
```
Comment (fgchaio) [inline DopplerJobsServer/appsettings.development.json:53]: why is this?

**Comment by fgchaio** [comment]
🚀

**Review by bamaro-dev — COMMENTED** [review]
Great!

File: DopplerJobsServer/appsettings.development.json (hunk 1 above)
Comment (nreal) [reply DopplerJobsServer/appsettings.development.json:53]: you are right about the question It doesn't make sense, I revert this change

**Pull request merged**